docs/api-reference/widgets/overview.md

```diff
@@ -4,10 +4,21 @@ Widgets are UI components around the WebGL2/WebGPU canvas to offer controls and
 
 This module contains the following widgets:
 
-- [FullscreenWidget](./fullscreen-widget.md)
+### Navigation Widgets
 - [ZoomWidget](./zoom-widget.md)
+- [ResetViewWidget](./reset-view-widget.md)
+
+### Geospatial Widgets
 - [CompassWidget](./compass-widget.md)
+<!-- - [ScaleWidget](./scale-widget.md) -->
+<!-- - [GeolocateWidget](./geolocate-widget.md) -->
 
+### Utility Widgets
+- [FullscreenWidget](./fullscreen-widget.md)
+- [ScreenshotWidget](./screenshot-widget.md)
+- [LoadingWidget](./loading-widget.md)
+- [ThemeWidget](./theme-widget.md)
+- [InfoWidget](./info-widget.md)
```

Comment:
Are we going with underscores on experimental widgets or not? Make a decision and update code and docs accordingly?
```suggestion
- [_InfoWidget](./info-widget.md)
```

Reply:
I think not to match the existing pattern: http://deck.gl/docs/api-reference/core/sun-light